### PR TITLE
Withdraw to a crypto address.

### PR DESCRIPTION
**Pull request by passabilities**

### CODE REVIEW

**Comment by fb55** [comment]
Could you add a test and document it in the README?

File: README.md [hunk 1]
```diff
@@ -406,6 +406,14 @@ const withdrawParamsBTC = {
   'coinbase_account_id': '536a541fa9393bb3c7000023', // BTC Coinbase Account ID
 };
 authedClient.withdraw(withdrawParamsBTC, callback);
+
+// Withdraw from your Exchange BTC account to another BTC address.
+const withdrawAddressParams = {
+  'amount': 10.00,
+  'currency': 'BTC',
+  'crypto_address': '1CQ4V3zij8v8BxHEEGQnTshJfW2ZRTEXes'
```
Comment (fb55) [inline README.md]: Please replace this with something that's not a valid address.

Comment (passabilities) [inline README.md]: ok i changed it

**Comment by boshdmg** [comment]
Was just going to do this myself :), does it really need a separate functions though? Could just handle it in the one withdraw method with different validation based on the params?

**Comment by fb55** [comment]
This looks good, only needs another rebase

**Comment by fb55** [comment]
Sorry, didn't see that this was rebased. Thanks a lot!

**Pull request merged**